.github/workflows/external-link-checker.yml

```diff
@@ -0,0 +1,36 @@
+# This code is a Qiskit project.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# This Action validates all the links in the documentation, including
+# external links and the validation of all the links in all the Qiskit
+# release notes. This Action will be run at 00:00 every day.
```

Comment:
It'd be good to mention a time zone. UTC, right?

Reply:
Yes, it's UTC time :+1: